### DISCUSSION TITLE
Nettoyer les descriptions.

### DISCUSSION THREAD

**Discussion by ghost** [friends-of-presta/fop_console#206] [Ideas]
Bonjour,

J'avance sur ma commande pour nettoyer les descriptions, mais je ne sais pas comment je vais faire la sortie.

Sur une petite boutique c'est facile ça donne ça : ( url 404 et `<p></p>` vide nettoyé ) 

![Capture d’écran de 2021-11-10 15-38-45](https://user-images.githubusercontent.com/5064590/141133809-87b7e20b-88ee-43b0-9fd0-44b324dc994f.png)

Mais sur une boutique avec 4-5k produits ça devient vite illisible.

Un CSV ? Juste les id_product ?

**Comment by SebSept** [comment]
j'afficherai le nombre de produits traités et modifiés simplement.
Pour aller plus loin, en mode verbeux, j'afficherai les ids.
Afficher le contenu, c'est bien juste pour debuger amha.
